Formula/black.rb

```diff
@@ -0,0 +1,45 @@
+class Black < Formula
+  include Language::Python::Virtualenv
+
+  desc "The uncompromising Python code formatter"
```

Comment:
`Python code formatter` is better: we want factual descriptions, not slogans :)

Reply:
Fair point, I'll fix that!

Reply:
@orn688 it looks like this line was not changed.

Reply:
Oops! Not sure how I messed that up. Thanks for catching it!